scripts/fast-reboot-dump.py

```diff
@@ -199,14 +204,37 @@ def send_arp(s, src_mac, src_ip, dst_mac_s, dst_ip_s):
 
     return
 
+def send_vlan_arp(s, src_mac, src_ip, dst_mac_s, dst_ip_s, vlan_iface):
```

Comment:
I think it's better to make one function to send an arp, but with argument dot1Qtag=False by default.

Reply:
Done